dwave/cloud/auth/flows.py

```diff
@@ -202,3 +222,72 @@ def from_config_model(cls, config: ClientConfig, **kwargs) -> LeapAuthFlow:
             authorization_endpoint=authorization_endpoint,
             token_endpoint=token_endpoint,
             session_config=session_config)
+
+    def run_oob_flow(self):
+        """Run the OAuth 2.0 Authorization Code exchange flow, using the out of
+        band code exchange.
+
+        After authorizing access by visiting the authorization URL displayed,
+        user has to copy the authorization code and paste it back in their
+        terminal. On successful completion Leap API access and refresh tokens
+        are returned.
+        """
+        self.redirect_uri = self._OOB_REDIRECT_URI
+
+        url = self.get_authorization_url()
+        print(self._VISIT_AUTH_MSG, url)
+
+        code = input(self._INPUT_CODE_MSG)
+        return self.fetch_token(code=code)
+
+    def run_redirect_flow(self, *, open_browser: Union[bool,Callable] = False,
+                          timeout: Optional[float] = None):
+        """Run the OAuth 2.0 Authorization Code exchange flow, using a redirect
+        URI hosted on a local server.
+
+        This flow is the preferred one, but can only be used if you can access
+        localhost addresses serviced by Ocean running in your (development)
+        environment from your browser.
+
+        After authorizing access by visiting the authorization URL displayed,
+        the flow will continue on the localhost redirect URI. The authorization
+        code is exchanged automatically, and on successful completion, Leap API
+        access and refresh tokens are returned.
```

Comment:
Would be very helpful to explain what access and refresh tokens are, or to link to the OAuth docs

Reply:
There is a link to OAuth 2.0 spec in `AuthFlow` docs, where these are explained I believe.